contents/docs/error-tracking/_snippets/nuxt-install-error-tracking.mdx

```diff
@@ -0,0 +1,91 @@
+import CLIUpload from "./cli/upload.mdx"
+import CLIAuthenticate from "./cli/authenticate.mdx"
+import CLIDownload from "./cli/download.mdx"
+
+## Installation
+
+Follow instructions [here](/docs/libraries/nuxt-js) to install `posthog-js` in your project.
+
+### Capturing exceptions
+
+#### 1. Manually
+
+Send errors directly using the posthog client, follow instructions on how to get the posthog instance.
+
+```vue component.vue
+<script>
+  function captureException() {
+    posthog.captureException(new Error("Error message"), {
+      ...additionalProperties
+    })
+  }
+</script>
+```
+
+#### 2. Automatically
+
+Update your `posthog.client.js` to add an error hook
+
+```js
+export default defineNuxtPlugin((nuxtApp) => {
+    ...
+    nuxtApp.hook('vue:error', (error) => {
+        posthogClient.captureException(error)
+    })
+    ...
+})
+```
+
+### Sourcemap upload
+
+#### 1. Download CLI
+
+<CLIDownload />
+
+#### 2. Authenticate
+
+<CLIAuthenticate />
+
+#### 3. Update your `nuxt.config.js`
+
+```js file=nuxt.config.js
+export default defineNuxtConfig({
+  ...,
+  sourcemap: {
+    client: true
+  },
+  hooks: {
+    'nitro:build:public-assets': async () => {
+        console.log('Running PostHog sourcemap injection...')
+        try {
+            execSync("posthog-cli sourcemap inject --directory '.output/public'", {
+                stdio: 'inherit',
+            })
+            console.log('PostHog sourcemap injection completed successfully')
+        } catch (error) {
+            console.error('PostHog sourcemap injection failed:', error)
+        }
+    },
+  }
+})
+```
+
+#### 4. Build your project for production
+
+```bash
+nuxt build
+```
+
+#### 5. Upload sourcemaps
+
+<CLIUpload />
+
+Where `./path/to/assets` is the path to your built assets. This is usually `.output/public` for Nuxt 3.
+
+#### 6. Delete sourcemaps (recommended)
```

Comment:
I feel like this step could confuse people. If you have public source maps disabled the previous steps presumably won't work, if they are enabled then this step is probably not necessary